catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v2_0_0/catalog/source/reader/FeatureCollectionMessageBodyReaderWfs20.java

```diff
@@ -176,7 +176,7 @@ public Wfs20FeatureCollection readFrom(
         throw new WebApplicationException(e1, response);
       } catch (JAXBException | XMLStreamException e1) {
         LOGGER.debug("Error in retrieving feature collection.", e1);
-      } catch (RuntimeException | Error e) {
+      } catch (Exception e) {
```

Comment:
❓ Are we sure that this is equivalent? I know it is not a good idea to catch Errors and RuntimeExceptions but there was most likely a reason for that. Further more we are now throwing back Exception which is not declared. I believe we could keep the original code as long as we had a 'catch (VirtualMachineError)' and throw it back right away without attempting to log